### PR TITLE
LPS-161019 Enable Account Administrators to edit basic user info for users who have valid email addresses

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-161019](https://issues.liferay.com/browse/LPS-161019).<br/><br/>/cc @pei-jung @ChrisKian @patriciajeanperez @epark18 @david-truong @gislayne-vitorino @RafaelCRC

### CODE REVIEW

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Hi @drewbrokke, one of tests will fail due to a test fix that I can forward with this pull after reviewing the rest of the results. I saw that we are now validating upper case domains, but wanted to confirm if we validate upon entering the upper case value in the modal and it will display as lowercase on the table? or it is a limitation that it will only display as lowercase once you save the account entry. Here is a GIF to explain myself better 😅 
0ma5Jlitum

**Comment by drewbrokke** [comment]
@patriciajeanperez it will only lowercase upon saving.

**Comment by drewbrokke** [comment]
@patriciajeanperez However, if it will make things easier, it is possible to lowercase things on the frontend before submission.

File: modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/validator/test/AccountEntryEmailValidatorTest.java [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+
+			accountEntryEmailValidatorExceptionUnsafeConsumer.accept(
+				_accountEntryEmailValidatorFactory.create(
+					_companyId, validatorArgs.validDomains));
+		}
+		finally {
+			ConfigurationTestUtil.saveConfiguration(
+				configuration, oldProperties);
+		}
+	}
```
Comment (pei-jung) [inline modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/validator/test/AccountEntryEmailValidatorTest.java:295]: Can we use `CompanyConfigurationTemporarySwapper` here?

Comment (drewbrokke) [reply modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/validator/test/AccountEntryEmailValidatorTest.java:295]: Oh good catch! I'll do that.

**Comment by drewbrokke** [comment]
@patriciajeanperez I made the change to lowercase on the frontend. Can you let me know if that fixes the issue?

**Comment by drewbrokke** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]
LGTM! Thanks for all the work, Drew!

**Comment by melodywu** [comment]
ci:test:account

**Comment by melodywu** [comment]
ci:test:bundle

**Comment by patriciajeanperez** [comment]
@drewbrokke Thanks, Drew! It should still fail because I need to update the assertion to lowercase, since we have a test that was adding uppercase and asserting the same value. If this PR is good to go and there are no more failures, I can forward this manually with the test fix/es.

**Comment by patriciajeanperez** [comment]
Hey @drewbrokke, Eric was able to reproduce the issue on his machine following these steps
# Add a virtual instance
# Go to virtual instance - instance settings - enable domain validation
# Add an account entry

The domain section does not display. I cannot reproduce it on my windows machine but I wonder if you can reproduce it on yours.

**Comment by drewbrokke** [comment]
Thanks @patriciajeanperez! Just pushed a fix for that issue. Please let me know if that solves it.

**Comment by drewbrokke** [comment]
@patriciajeanperez just pushed a change that removes the account domains config from System Settings. Are there any test cases that use System Settings for this, or is it all Instance Settings?

**Comment by patriciajeanperez** [comment]
Thanks @drewbrokke! We have 1 test that touches System Settings. I will send a new PR with the test fixes.

**Comment by patriciajeanperez** [comment]
Resent https://github.com/liferay-usm/liferay-portal/pull/724

**Comment by patriciajeanperez** [comment]
ci:close

**Pull request closed without merging**